src/components/services/PiHole.vue

```diff
@@ -37,6 +37,9 @@ export default {
     ads_percentage_today: 0,
   }),
   computed: {
+    apiQuery() {
+      return this.item.apikey ? "`?summaryRaw&auth=${this.item.apikey}`" : "";
```

Comment:
```suggestion
      return this.item.apikey ? `?summaryRaw&auth=${this.item.apikey}` : "";
```
backticks are the same as doublequote + string interpolation. it replace the doublequote.